package.json

```diff
@@ -34,32 +34,33 @@
     "access": "public"
   },
   "peerDependencies": {
-    "@callstack/repack": "blitzstudios/repack.git#callstack-repack-v2.6.3-gitpkg",
+    "@callstack/repack": "2.6.3",
```

Comment:
Just thinking aloud here - maybe it's time that we actually publish our forks of these projects on npm - that way we don't have to worry about version incompatibilities by devs not using the right branches. This is a bit ambiguous - both our forked gitpkg and the official 2.6.3 release would match.

Reply:
We probably should, but maybe when things settle a bit?